components/edit_post_modal/edit_post_modal.tsx

```diff
@@ -92,30 +152,33 @@ class EditPostModal extends React.PureComponent {
         document.removeEventListener('paste', this.handlePaste);
     }
 
-    setShowPreview = (newPreviewValue) => {
+    setShowPreview = (newPreviewValue: boolean) => {
         this.props.actions.setShowPreview(newPreviewValue);
-    }
+    };
 
     getContainer = () => {
         return this.editModalBody.current;
-    }
+    };
 
     toggleEmojiPicker = () => {
         this.setState({showEmojiPicker: !this.state.showEmojiPicker});
         if (!this.state.showEmojiPicker && this.editbox) {
             this.editbox.focus();
         }
-    }
+    };
 
     hideEmojiPicker = () => {
         this.setState({showEmojiPicker: false});
         if (this.editbox) {
             this.editbox.focus();
         }
-    }
+    };
 
-    handleEmojiClick = (emoji) => {
-        const emojiAlias = emoji && (emoji.name || (emoji.aliases && emoji.aliases[0]));
+    handleEmojiClick = (emoji: Emoji | null) => {
```

Comment:
Unless we're explicitly checking for `null`, we can just make this an optional parameter
```suggestion
    handleEmojiClick = (emoji?: Emoji) => {
```